game/scripts/vscripts/lib/util.ts

```
@@ -467,3 +466,31 @@ function CanUnitCastAbility(caster: CDOTA_BaseNPC, ability: CDOTABaseAbility, ta
 		) === UnitFilterResult.SUCCESS
 	);
 }
+
+/**
+ * Get the dota team for a sepcific index.
+ * @param index team index (1-8)
+ * @returns according dota team
+ */
+export function GetTeamFromIndex(index: number): DotaTeam {
+	switch (index) {
+		case 1:
+			return DotaTeam.CUSTOM_1;
+		case 2:
+			return DotaTeam.CUSTOM_2;
+		case 3:
+			return DotaTeam.CUSTOM_3;
+		case 4:
+			return DotaTeam.CUSTOM_4;
+		case 5:
+			return DotaTeam.CUSTOM_5;
+		case 6:
+			return DotaTeam.CUSTOM_6;
+		case 7:
+			return DotaTeam.CUSTOM_7;
+		case 8:
+			return DotaTeam.CUSTOM_8;
+		default:
+			return DotaTeam.NOTEAM;
```

This should never occur, and therefore should throw an error if this happens.